src/lib/storage/refinements/partition_attributes.rb

```diff
@@ -42,9 +42,9 @@ def label
           end
 
           # @return [DiskSize]
-          def size
-            DiskSize.KiB(size_k)
-          end
+#          def size
+#            DiskSize.KiB(size_k)
+#          end
 
```

Comment:
Now `partition.size` returns a Fixnum (bytes) isn't it? In that case, instead of commenting this method I would prefer to rename it to `disk_size` or something similar.